tendrl/commons/event.py

```diff
@@ -0,0 +1,25 @@
+from gevent import socket
+from gevent.socket import error as socket_error
+from gevent.socket import timeout as socket_timeout
+from tendrl.commons.message import Message
+
+
+class EventFailed(Exception):
+    pass
+
+
+class Event(object):
+    def __init__(self, message):
+        self.sock = socket.socket(socket.AF_UNIX, socket.SOCK_STREAM)
+        self._write(message)
+
+    def _write(self, message):
+        try:
+            socket_path = tendrl_ns.config.data['logging_socket_path']
+            self.sock.connect(socket_path)
+            json_str = Message.to_json(message)
+            self.sock.send(json_str)
+            self.sock.close()
+        except (socket_error, socket_timeout,
+                TypeError) as ex:
+            raise EventFailed(ex)
```

Comment:
Here we are raising the event if the socket fails. How should this situation be handled ? Should the caller of the function should handle this every time he logs ? @nthomas-redhat @shtripat please review